src/main.cc

```diff
@@ -33,8 +33,6 @@
 #if defined( USE_BREAKPAD )
   #if defined( Q_OS_MAC )
     #include "client/mac/handler/exception_handler.h"
-  #elif defined( Q_OS_LINUX )
-    #include "client/linux/handler/exception_handler.h"
   #elif defined( Q_OS_WIN32 )
```

Comment:
why remove this

Reply:
Because we are not Google.

Google can hire dedicated build engineer(s) to build a mostly statistically linked Chrome. So, they can analyze the breakpad dumps based on their build's debug info.

We don't have that kind of resource. Everyone's GD on Linux are slightly different (ranging from different distros, libs, compiler configurations.......). So the breakpad support is useless.

Reply:
The application's debug info should be enough for analysis.

Reply:
What debug info? Where are they coming from? Think before talking.

Reply:
```
-DCMAKE_BUILD_TYPE=RelWithDebInfo
```

try to learn how to respect others. maybe I give you too much right.

Reply:
If someone builds GD from source that way, there are many easy and common ways to get backtrack.

In all other places, like almost all distros' repos, the debug symbols will be stripped. The doc that tells users to use breakpad is misleading and waste of time.

I already told you "Everyone's GD installation is different". Breakpad is hardly useful because the debug info doesn't even exist in 99% of the situations.

Replace the related doc and delete the code is the best course of action.